PyInstaller/hooks/hook-usb.py

```diff
@@ -64,24 +78,19 @@
         'openusb',
     )
 
+    backend_library_basenames = []
     for candidate in libusb_candidates:
         libname = ctypes.util.find_library(candidate)
         if libname is not None:
-            break
-
-    if libname is not None:
-        # Use basename here because Python returns full library path
-        # on Mac OSX when using ctypes.util.find_library.
-        bins = [os.path.basename(libname)]
-        binaries = _resolveCtypesImports(bins)
-    elif is_cygwin:
-        bins = ['cygusb-1.0-0.dll', 'cygusb0.dll']
-        binaries = _resolveCtypesImports(bins)[:1] # use only the first one
-    else:
-        binaries = []
-
-    if binaries:
-        # `_resolveCtypesImports` returns a 3-tuple, but `binaries` are only
-        # 2-tuples, so remove the last element:
-        assert len(binaries[0]) == 3
-        binaries = [(binaries[0][1], '')]
+            backend_library_basenames.append(os.path.basename(libname))
+
+    if backend_library_basenames:
+        binaries = _resolveCtypesImports(backend_library_basenames)
+
+
+# However a supported usb library was found, validate it and normalize it.
```

Comment:
"However" is the wrong word, it means something like "but", "yet". IMHO this should be "No matter how"